### DISCUSSION TITLE
Guys , is Anny one here noticed the issue in the documentation or im wrong ?

### DISCUSSION THREAD

**Discussion by ahmedbou2000** [QuestPDF/QuestPDF#946] [General]
i have seen an issue in the documentation of questPDF and i want to ask if any one noticed that before ! 
https://www.questpdf.com/api-reference/table.html#column-definitions
![QuestPDF Bug](https://github.com/QuestPDF/QuestPDF/assets/109955655/5e8eedee-accb-4023-a7e1-50bcec8384a2)

**Comment by girlpunk** [comment]
This screenshot has been highlighted as problematic before (https://github.com/QuestPDF/QuestPDF/discussions/760), I wonder if the code and screenshot both warrant redoing at this point?

**Reply by ahmedbou2000** [reply]
Thank you for acknowledging the issue. As the one who previously reported it, I noticed that the documentation and example have not been updated yet. Could you please provide an estimated timeline for when this correction will be made? Additionally, any interim clarification on how relative columns are calculated would be greatly appreciated.

**Comment by MarcinZiabek** [comment]
Thank you for letting us know about the issue. I have just released a fix. https://www.questpdf.com/api-reference/table.html#column-definitions

**Reply by ahmedbou2000** [reply]
Thank you MarcinZiabek .
